lib/libutils/ext/sub.mk

```diff
@@ -9,5 +9,5 @@ srcs-y += nex_strdup.c
 srcs-y += consttime_memcmp.c
 srcs-y += memzero_explicit.c
 
-subdirs-$(arch_arm) += arch/$(ARCH)
+subdirs-$(arch_$(ARCH)) += arch/$(ARCH)
```

Comment:
There is another instance in `lib/libutils/isoc/sub.mk`. Besides, I am wondering if that shouldn't be `subdirs-y += arch/$(ARCH)`. What does `$(arch_$(ARCH))` bring us?

Reply:
Updated `lib/libutils/isoc/sub.mk` also. I'm not sure if there was any reason to make it like `$(arch_$(ARCH))`. But changed to `subdirs-y += arch/$(ARCH)` as you suggested.